libvast/src/msgpack_table_slice.cpp

```diff
@@ -326,8 +326,8 @@ data_view msgpack_table_slice<FlatBuffer>::at(table_slice::size_type row,
   auto view = as_bytes(*slice_.data());
   // First find the desired row...
   VAST_ASSERT(row < offset_table.size());
-  VAST_ASSERT(state_.layout.at(*state_.layout.offset_from_index(column))->type
-              == t);
+  VAST_ASSERT(congruent(
+    state_.layout.at(*state_.layout.offset_from_index(column))->type, t));
```

Comment:
Note to the reviewer: This change was a bug fix. The old assertion needlessly triggered for aliases and types with attributes.